frontend/src/scenes/authentication/signup/signupForm/test/panels/SignupPanel2.tsx

```diff
@@ -0,0 +1,106 @@
+import { LemonInput, LemonSelect, LemonButton, Link } from '@posthog/lemon-ui'
+import { useValues } from 'kea'
+import { Form } from 'kea-forms'
+import { Field } from 'lib/forms/Field'
+import RegionSelect from 'scenes/authentication/RegionSelect'
+import { preflightLogic } from 'scenes/PreflightCheck/preflightLogic'
+import { signupLogic } from '../signupLogic'
+
+const UTM_TAGS = 'utm_campaign=in-product&utm_tag=signup-header'
+
+export function SignupPanel2(): JSX.Element | null {
+    const { preflight } = useValues(preflightLogic)
+    const { isSignupPanel2Submitting } = useValues(signupLogic)
+
+    return (
+        <div className="space-y-4 Signup__panel__2">
+            <Form logic={signupLogic} formKey={'signupPanel2'} className="space-y-4" enableFormOnSubmit>
+                <RegionSelect />
+                <Field name="first_name" label="Your name">
+                    <LemonInput
+                        className="ph-ignore-input"
+                        data-attr="signup-first-name"
+                        placeholder="Jane Doe"
+                        disabled={isSignupPanel2Submitting}
+                    />
+                </Field>
+                <Field name="organization_name" label="Organization name">
+                    <LemonInput
+                        className="ph-ignore-input"
+                        data-attr="signup-organization-name"
+                        placeholder="Hogflix Movies"
+                        disabled={isSignupPanel2Submitting}
+                    />
+                </Field>
+                <Field name="role_at_organization" label="What is your role?">
+                    <LemonSelect
```

Comment:
If the role is `other`, could we add an input to ask which role? Maybe inlined to this select.
If too ugly or bothersome to add, we can skip for now.

Reply:
Let's skip for now and see if having the extra data is useful for anyone.